src/dev-tools/dev-tools.js

```diff
@@ -128,7 +128,6 @@ function install() {
               height: '60px',
               width: '68px',
               transition: 'all 0.3s',
-              overflow: 'scroll',
```

Comment:
We should add `overflow: 'scroll'` to the styles when the dev tools are visible, otherwise small screens won't be able to see everything.